### ISSUE TITLE
selection_filters passes on local machine, fails on ecocloud

### ISSUE THREAD

**Issue by shawnlaffan** [AtlasOfLivingAustralia/galah#43]
I have this section of code below, adapted from [the docs](https://atlasoflivingaustralia.github.io/galah/articles/galah.html#filtering-data-1).

I am using galah 1.1.0

```R
selection_filters <- select_filters(
    basisOfRecord = "HumanObservation",
    profile = "ALA")
```

On my local machine (windows 10, R 4.0.3) it runs without issue.  However if I run it on a jupyter notebook via the ecocloud.org.au system (linux, R 3.6.1) then I get this error:

```
Error in validate_filters(filters) : 
  The following filters are invalid: basisOfRecord. Use `search_fields()` to get a list of valid options
```

This is possibly related to, or a duplicate of, #38 but I have not checked the internals to verify. 

Shawn.

**Comment by matildastevenson** [comment]
Hi Shawn,
#38 has been fixed in the `dev` branch- could you install the development version of `galah` to see if you still get the error?

**Comment by shawnlaffan** [comment]
This issue seems not to be fixed by the fix for #38.  

I updated the code and ran the snippet, receiving the same error.  

That of course assumes I have installed the branch correctly.

A student had the same issue today on their local machine when using 3.6, so it appears not to be an ecocloud/conda problem.

There is also an error with the taxa argument to ala_occurrences under 3.6 which I will raise as a separate issue. (throws error ```Error: taxa is not a character vector```)
